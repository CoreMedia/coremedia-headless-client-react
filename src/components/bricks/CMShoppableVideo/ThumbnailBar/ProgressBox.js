// @flow
import styled from 'styled-components';

const ProgressBox = styled.div`
  width: 100%;
`;
ProgressBox.displayName = 'ProgressBox';

export default ProgressBox;
