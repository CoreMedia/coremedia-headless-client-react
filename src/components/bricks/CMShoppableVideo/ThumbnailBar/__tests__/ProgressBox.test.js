// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import ProgressBox from '../ProgressBox';

describe('ProgressBox Component', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<ProgressBox>Content</ProgressBox>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
