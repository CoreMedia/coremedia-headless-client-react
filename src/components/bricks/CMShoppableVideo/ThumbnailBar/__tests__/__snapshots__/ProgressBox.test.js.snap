// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBox Component should render correctly 1`] = `
.c0 {
  width: 100%;
}

<div
  className="c0"
>
  Content
</div>
`;
